.storybook/components/Docs/Guildelines/Documents.stories.tsx

```diff
@@ -0,0 +1,53 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+import Code from '../../../../docs/CODE_GUIDELINES.md';
+import ComponentDocs from '../../../../docs/COMPONENTS.md';
+import IconDocs from '../../../../docs/ICONS.md';
+import LayoutDocs from '../../../../docs/LAYOUT.md';
+import TokenDocs from '../../../../docs/TOKENS.md';
+import TypographyDocs from '../../../../docs/TYPOGRAPHY.md';
+import { Documentation } from '../Documentation';
+
+export default {
+  title: 'Documentation/Guidelines',
+  component: Documentation,
+  parameters: {
+    chromatic: { disableSnapshot: true },
+    axe: {
+      skip: true,
+    },
+  },
+  decorators: [
+    (Story) => (
+      <Documentation>
+        <Story />
+      </Documentation>
+    ),
+  ],
```

Comment:
this could be a JSON in Docs/ that gets imported into each of the subsequent documentation pages, so that they all share the same grammar. DRY-ing the story config some

Reply:
Oh good idea. Placed it in the `MarkdownWrapper.tsx` file instead tho since it needs a tsx extension for the decorator